molo/profiles/models.py

```diff
@@ -52,19 +65,33 @@ class UserProfilesSettings(BaseSetting):
                 FieldPanel('mobile_number_required'),
                 FieldPanel('prevent_phone_number_in_username'),
             ],
-            heading="Mobile Number Settings",),
+            heading="Mobile Number Settings", ),
         MultiFieldPanel(
             [
                 FieldPanel('show_email_field'),
                 FieldPanel('email_required'),
                 FieldPanel('prevent_email_in_username'),
             ],
-            heading="Email Settings",)
+            heading="Email Settings", ),
+        MultiFieldPanel(
+            [
+                FieldPanel("show_security_question_fields"),
+                FieldPanel("security_questions_required"),
+            ],
+            heading="Security Question Settings", )
     ]
     # TODO: mobile_number_required field shouldn't be shown
     # if show_mobile_number_field is False
 
 
+class SecurityQuestion(models.Model):
```

Comment:
Incase you're not aware - in order to allow admins to create these types of models, we use wagtail's [ModelAdmin](https://github.com/rkhleics/wagtailmodeladmin) for adding the create/edit views in Wagtail. See [example in molo.core](https://github.com/praekelt/molo/blob/develop/molo/core/wagtail_hooks.py#L19-L30)